test/unit/UFragmentsPolicy.js

```diff
@@ -423,9 +487,45 @@ contract('UFragmentsPolicy:Rebase', async function (accounts) {
 contract('UFragmentsPolicy:Rebase', async function (accounts) {
   before('setup UFragmentsPolicy contract', setupContracts);
 
-  describe('rate=1', function () {
+  describe('with inflation', function () {
```

Comment:
This test case name is really confusing, since inflation could refer to USD being devalued, CPI increasing, or Ample supply increasing.

How about 'when cpi increases'?

Same with the 'deflation' case below.

Reply:
👍